modules/installation-process.adoc

```diff
@@ -3,41 +3,42 @@
 // * installing/index.adoc
 // * architecture/architecture-installation.adoc
 
+:_content-type: CONCEPT
 [id="installation-process_{context}"]
 = Installation process
 
-Except for the {ai-full}, when you install an {product-title} cluster, you download the installation program from
+Except for the {ai-full}, when you install an {product-title} cluster, you must download the installation program from
 ifndef::openshift-origin[]
-the appropriate link:https://console.redhat.com/openshift/install[Infrastructure Provider] page on the {cluster-manager} site. This site manages:
+the appropriate link:https://console.redhat.com/openshift/create[*Cluster Type*] page on the {cluster-manager} {hybrid-console-second}. This console manages:
 
-* REST API for accounts
-* Registry tokens, which are the pull secrets that you use to obtain the required components
-* Cluster registration, which associates the cluster identity to your Red Hat account to facilitate the gathering of usage metrics
+* REST API for accounts.
+* Registry tokens, which are the pull secrets that you use to obtain the required components.
+* Cluster registration, which associates the cluster identity to your Red Hat account, so to facilitate the gathering of usage metrics
```

Comment:
I think adding "so" here makes this more confusing to read imo, I'd personally keep it the way it was and add a period for parallel structure:

```suggestion
* Cluster registration, which associates the cluster identity to your Red Hat account to facilitate the gathering of usage metrics.
```